### ISSUE TITLE
Miner blocked by cobblestone

### ISSUE THREAD

**Issue by kd5crs** [ldtteam/minecolonies#5483]
# Prerequisites

- [ ] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [X ] I checked the MineColonies [Wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [X ] I made sure that **this issue is not a duplicate of any existing issue.**
- [X ] This is an issue and not a feature request. Please create all feature requests [here](https://github.com/ldtteam/minecolonies-features/issues).

### Context

- Minecraft Version:1.15.2
- MineColonies Version:minecolonies-0.13.78-ALPHA-universal
- Related Mods and Their Versions: 

### Expected behavior

The miner should mine through blocks between it and its destination when in the mine. If it can't get where it is going, it should go try a different node.

### Actual behaviour

I found my miner stuck trying to go down a hallway it had dug in the mine. Somehow, either by lava/water interaction or by the miner placing it, the 3x3 hallway was blocked by cobblestone. If I hit the miner, they would run off but come back to the same spot, clearly trying to walk through the cobblestone. No errors came up, the miner would go to bed at the end of the workday, and then upon waking go back to trying to walk through the cobblestone.

As soon as I mined the cobblestone blocking the hallway, the miner went down the hallway to the end and re-started mining like normal. 

### Steps to reproduce the issue

1. Place a miner and let them start mining
2. When you know what branch they are going down, block it with cobblestone (maybe), or use lava/water to block it with cobblestone (maybe)
3. See miner become standarounder
4. ... Profit

### Logs

- latest.log: https://gist.github.com/kd5crs/5aa216bfaa8ef5431843ed3bf5bc9f45
- crashlog: no crash

**Comment by lebanni** [comment]
👍

**Comment by leagris** [comment]
Still an issue a year later.

See video:

[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/MLnY2VPYSPs/0.jpg)](https://youtu.be/MLnY2VPYSPs)

**Comment by Raycoms** [comment]
Miner fluid handling was improved
